### PR TITLE
add index.js with clickable answer buttons

### PR DESCRIPTION
**Pull request by flexii07**

### CODE REVIEW

File: index.html
Comment (sebastianwel) [inline index.html:1]: lumos! --> sehr geil 👍

File: index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const button = document.querySelector('.answer1');
+    const answer = document.querySelector('#answer1');
+    const button2 = document.querySelector('.answer2');
+    const answer2 = document.querySelector('#answer2');
+    const bookmarkLink = document.getElementById('bookmark-link');
+    const bookmarkIcon = document.querySelector('.bookmark-icon');
```
Comment (dimitriosxmi) [inline index.js:7]: interessant approach awesome!

**Review by sebastianwel — APPROVED** [review]
Looks good to me so far, 10 Points for Gryffindor! Really like the HP design :D

**Review by dimitriosxmi — APPROVED** [review]
## LGTM
Note: Like the approach on accessing classes and id's through the query selector, just learned something new! thanks 😃!

**Comment by flexii07** [comment]
> ## LGTM
> Note: Like the approach on accessing classes and id's through the query selector, just learned something new! thanks 😃!

just a thank you for the help yesterday haha

**Pull request merged**